client/ayon_core/tools/publisher/widgets/product_attributes.py

```diff
@@ -121,8 +170,14 @@ def _refresh_content(self):
             col_num = 2 - expand_cols
 
             label = None
+            is_overriden = False
             if attr_def.is_value_def:
+                is_overriden = any(
+                    item["value"] != item["default"]
```

Comment:
I wonder if it's worth having just a `is_default()` function on the info items so we can encapsulate the logic there?